scripts/core/editor3/reducers/suggestions.jsx

```diff
@@ -374,8 +375,8 @@ const pasteAddSuggestion = (state, {content, data}) => {
     // store current selection for later
     const finalSelection = editorState.getSelection();
     const newSelection = initialSelection.merge({
-        anchorKey: initialSelection.getEndKey(),
```

Comment:
I think that only the new text should be set as add suggestion. The old text from getStartOffset to getEndOffset will be set as delete suggestion.

Reply:
yep it is, but it has to select if from start which is where initial selection started, old text is pushed further by replaceWithFragment